Polyline.xcodeproj/project.pbxproj

```diff
@@ -395,6 +512,90 @@
 			};
 			name = Release;
 		};
+		91F7833E1CE4EB7F004E87E3 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_ANALYZER_NONNULL = YES;
+				CODE_SIGN_IDENTITY = "-";
+				COMBINE_HIDPI_IMAGES = YES;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEFINES_MODULE = YES;
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_VERSION = A;
+				GCC_NO_COMMON_BLOCKS = YES;
+				INFOPLIST_FILE = "$(SRCROOT)/Polyline/Info.plist";
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
```

Comment:
The podspec says 10.10 is the minimum Mac deployment target. Earlier versions almost certainly work too, but more importantly, the project should be consistent with the podspec:

1. In the project editor, select the Polyline project (the item above the targets). Go to the Info tab and change OS X Deployment Target to 10.10.
1. Select the PolylineMac target and switch to the Build Settings tab. Select the “OS X Deployment Target” row and hit Delete. Do the same for the PolylineMacTests target. This way the deployment target is defined in one place.